docker/py-dsci-100/Dockerfile

```diff
@@ -15,11 +15,6 @@ WORKDIR "${HOME}/work"
 # https://discourse.jupyter.org/t/debugger-warning-it-seems-that-frozen-modules-are-being-used-python-3-11-0/16544/12
 ENV PYDEVD_DISABLE_FILE_VALIDATION=1
 
-# update scikit-learn to the current @main (July 20, 2023) 
-# to get this merged PR: https://github.com/scikit-learn/scikit-learn/pull/26772
-# which fixes this issue: https://github.com/scikit-learn/scikit-learn/issues/26768
-RUN pip install --upgrade git+https://github.com/scikit-learn/scikit-learn@7de59b2017f39048880a3858d32c8caef9308954
```

Comment:
What installs scikit-learn now that this is gone?

Reply:
(and does the latest release include the two fixes listed in the comments?)

Reply:
scikit-learn 1.3.1 is installed in the base notebook `jupyter/scipy-notebook`. Here's a link to the [relevant patch notes for 1.3.1](https://scikit-learn.org/stable/whats_new/v1.3.html#sklearn-neighbors), fixed by PR #26772.